x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -54,6 +54,10 @@ export function overriddenTimeBounds(state: DataState) {
   return state.overriddenTimeBounds;
 }
 
+export function agentId(state: DataState) {
```

Comment:
suggestion: properly type the function by adding a return type to it.

Reply:
Will do! Thank you @paul-tavares :)